db.go

```diff
@@ -124,7 +115,7 @@ func (m *DbMap) createIndexImpl(dialect reflect.Type,
 		s.WriteString(fmt.Sprintf(" %s %s", m.Dialect.CreateIndexSuffix(), index.IndexType))
 	}
 	s.WriteString(";")
-	_, err := m.Exec(s.String())
+	_, err := m.ExecContext(context.TODO(), s.String())
```

Comment:
Are these methods essentially broken until we put a proper context?
> TODO returns a non-nil, empty Context.
```
createIndexImpl
DropIndex
createTables
dropTableImpl
TruncateTables
```

Reply:
I've gone through and replaced these with pass-throughs for a real context. Note that the methods weren't broken, they were just getting the equivalent of `context.Background()`.

Reply:
Ah, I misinterpreted the doc text.